### PR TITLE
New OM: Type Illusion Meta (BETA)

### PR DESCRIPTION
**Pull request by HiTechFlufi**
Adding Type Illusion Beta!!

- flufi

### CODE REVIEW

**Comment by ghost** [comment]
@DeathlyPlays were is my collaborator

**Comment by ghost** [comment]
and the labels

**Comment by ghost** [comment]
we prolly need to test it so everything works

File: config/formats.js [hunk 1]
```diff
@@ -1414,6 +1414,110 @@ exports.Formats = [
...
+				pokemon.types = ["Bug", "Ghost"];
+			}
+			if (name === 'V!X3N' && !pokemon.illusion) {
+				this.add('-start', pokemon, 'typechange', 'Fire/Bug/Electric');
+				pokemon.types = ["Fire", "Bug", "Electric"];
+			}
+			if (name === 'flufi' && !pokemon.illusion) {
+				this.add('-start', pokemon, 'typechange', 'Normal/Ghost');
+				pokemon.types = ["Normal", "Ghost"];
+		},
```
Comment (ghost) [inline config/formats.js]: `unexpected token:  ,`

**Review by ghost — CHANGES_REQUESTED** [review]

File: config/formats.js [hunk 2]
```diff
@@ -1414,6 +1414,118 @@ exports.Formats = [
 		},
 	},
 	{
+		name: "Type Illusion [Beta]",
+		section: "Exiled's Custom Gamemodes",
+		mod: 'gen7',
```
Comment (DeathlyPlays) [inline config/formats.js]: There is no Gen 7 if you meant a gen 7 battle add [Gen 7] in front of the name like 

> name: "[Gen 7] Type Illusion [BETA]",

**Review by DeathlyPlays — CHANGES_REQUESTED** [review]

**Review by DeathlyPlays — APPROVED** [review]

**Pull request merged**